google-cloud-spanner/src/test/java/com/google/cloud/spanner/InstanceAdminClientImplTest.java

```diff
@@ -294,21 +314,26 @@ public void testCreateInstanceWithProcessingUnits() throws Exception {
   }
 
   @Test
-  public void testCreateInstanceWithBothNodeCountAndProcessingUnits() throws Exception {
-    try {
-      client.createInstance(
-          InstanceInfo.newBuilder(InstanceId.of(PROJECT_ID, INSTANCE_ID))
-              .setInstanceConfigId(InstanceConfigId.of(PROJECT_ID, CONFIG_ID))
-              .setNodeCount(1)
-              .setProcessingUnits(100)
-              .build());
-      fail("missing expected exception");
-    } catch (IllegalArgumentException e) {
-      assertTrue(
-          e.getMessage()
-              .contains(
-                  "Only one of nodeCount and processingUnits can be set when creating a new instance"));
-    }
+  public void testCreateInstanceWithAutoscalingConfig() throws Exception {
+    OperationFuture<com.google.spanner.admin.instance.v1.Instance, CreateInstanceMetadata>
+        rawOperationFuture =
+            OperationFutureUtil.immediateOperationFuture(
+                "createInstance",
+                getAutoscalingInstanceProto(),
+                CreateInstanceMetadata.getDefaultInstance());
+    when(rpc.createInstance("projects/" + PROJECT_ID, INSTANCE_ID, getAutoscalingInstanceProto()))
+        .thenReturn(rawOperationFuture);
+    OperationFuture<Instance, CreateInstanceMetadata> operation =
+        client.createInstance(
+            InstanceInfo.newBuilder(InstanceId.of(PROJECT_ID, INSTANCE_ID))
+                .setInstanceConfigId(InstanceConfigId.of(PROJECT_ID, CONFIG_ID))
+                .setAutoscalingConfig(getAutoscalingInstanceProto().getAutoscalingConfig())
+                .build());
+    assertTrue(operation.isDone());
```

Comment:
Nit: store operation.get() in a local object say `expectedOutput` and then use it in every `assertEquals` statement instead of doing operation.get() repeatedly.